packages/stencil/src/components/datacom-list/datacom-list.css

```diff
@@ -3,38 +3,29 @@
 }
 
 :host * {
-  font-family: var(--dc-font-family);
-}
-
-ol {
-  margin-bottom: 2px;
-  margin-top: 2px;
-}
-
-ul {
   margin-bottom: 2px;
   margin-top: 2px;
 }
 
-.list-ordered {
+.dc-list-ordered {
   font-size: 16px;
-  color: #0a1839;
+  color: var(--dc-font-family);
   list-style: decimal;
   font-weight: 400;
 }
 
-.list-unordered {
+.dc-list-unordered {
   list-style: disc outside none;
 }
 
-.list-lowercase {
+.dc-list-lowercase {
   list-style: lower-alpha;
 }
 
-.list-upercase {
+.dc-list-upercase {
```

Comment:
Misspelt 'upercase'. Should be 'uppercase'.

Reply:
Do remember to also update your usage.md file and your stories with the same spelling correction above.

Reply:
Done